src/qaqc/download_and_flag_fxns/make_threshold_table.R

```diff
@@ -1,25 +1,26 @@
 make_threshold_table <- function(df){
 
-  # df <- df %>%
-  #   mutate(mean = ifelse(grepl("outside of sensor specification range", flag), ) %>%
-  #   group_by(season) %>%
-  #   site = paste0(unique(site)),
-  #   parameter = paste0(unique(parameter)),
-  #   mutate(p01 = quantile(mean, 0.01, na.rm = TRUE),
-  #          p99 = quantile(mean, 0.99, na.rm = TRUE)) %>%
-  #   # need to pull some of this info out and use it for the range limits to test it
-  #   summarize(m_sd_0199 = sd(mean, na.rm = T)) %>%
-  #   select(season, m_sd_0199),
-  # by = "season")
+  good_data_stats <- df %>%
+    # REMOVE DATA WE KNOW TO BE ERRONEOUS:
+    add_malfunction_flag() %>%
+    filter(is.na(flag)) %>%
+    group_by(season) %>%
+    mutate(f01 = quantile(mean, 0.01, na.rm = TRUE),
+           f99 = quantile(mean, 0.99, na.rm = TRUE),
+           f_slope_behind_01 = quantile(slope_behind, 0.01, na.rm = TRUE),
+           f_slope_behind_99 = quantile(slope_behind, 0.99, na.rm = TRUE)) %>%
```

Comment:
question: how do these quantiles work now that slope behind is not an absolute value?

I don't know if I understand this correctly, but if we wanted to differentiate between slopes going and slopes going down I would think that we want to separate the positive values and the negative values into their own columns and then get the 99th quantiles from those categories? This stems from my understanding that the 1st percentile of slopes would be the smallest slopes, but a negative slope wouldn't necessarily mean a small slope in this instance, since we are no longer taking absolute values.